modules/compute/src/integrationTest/java/org/apache/ignite/internal/compute/events/ItComputeEventsEmbeddedTest.java

```diff
@@ -35,4 +48,53 @@ protected IgniteCompute compute() {
     protected EventMatcher jobEvent(IgniteEventType eventType, Type jobType, @Nullable UUID jobId, String jobClassName, String targetNode) {
         return embeddedJobEvent(eventType, jobType, jobId, jobClassName, targetNode, node(0).name());
     }
+
+    // Cancel tests are hard to implement without global signals, let's test them in the embedded mode only.
```

Comment:
I don't think there is any difference, we can use global signals the same way for client-initiated tasks.